pkg/detectors/rootly/rootly.go

```diff
@@ -0,0 +1,95 @@
+package rootly
+
+import (
+	"context"
+	"net/http"
+
+	// "strings"
+	"fmt"
+
+	"github.com/trufflesecurity/trufflehog/v3/pkg/common"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/detectors"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/pb/detectorspb"
+	regexp "github.com/wasilibs/go-re2"
+)
+
+type Scanner struct{}
+
+// Ensure the Scanner satisfies the interface at compile time.
+var _ detectors.Detector = (*Scanner)(nil)
+
+var (
+	client = common.SaneHttpClient()
+
+	// Make sure that your group is surrounded in boundary characters such as below to reduce false positives.
+	keyPat = regexp.MustCompile(`\brootly_[a-f0-9]{64}\b`)
```

Comment:
This looks good, but it might be more robust if we wrap the regex in a capturing group